{{cookiecutter.app_name}}/Makefile

```
@@ -43,8 +43,15 @@ install-dependencies:
 test:
 	docker-compose -f docker-compose.test.yml up -d
 	ENV=test make db/migrate
-	go test -v -p 1 -count=1 ./...
+	go test -v -p 1 -count=1 ./... -coverprofile=coverage/coverage.out
```

Should we also add test report file so that we can use it to publish the test result later on CI?
```suggestion
	go test -v -p 1 -count=1 ./... -coverprofile=coverage/coverage.out  -ginkgo.reportFile=coverage/test-report.xml
```

---

Added in [c8ed319](https://github.com/nimblehq/gin-templates/pull/95/commits/c8ed3196b0640ed63d9667f1e7ae45183d7ffb4f)